frontend/src/metabase/entities/users/forms.js

```diff
@@ -13,13 +13,15 @@ const getNameFields = () => [
     title: t`First name`,
     placeholder: "Johnny",
     autoFocus: true,
-    validate: validate.required().maxLength(100),
+    validate: validate.maxLength(100),
+    normalize: firstName => firstName || null,
```

Comment:
If we don't normalize the field, the value could go in either direction of
- If the field is touched, the value will be an empty string `""`
- If the field isn't touched, the value will be `null`